### PR TITLE
[core] remove noisy logs

### PR DESCRIPTION
**Pull request by degemer**
### What does this PR do?

Remove noisy logs.

### Motivation

Not see them anymore.

### Additional Notes

:fire:

### CODE REVIEW

File: daemon.py [hunk 1]
```diff
@@ -154,7 +154,6 @@ def daemonize(self):
 
 
     def start(self, foreground=False):
-        log.info("Starting")
```
Comment (olivielpeau) [inline daemon.py]: I like this line, it lets you search easily in the logs for when the component started.

**Comment by olivielpeau** [comment]
Just added one small 🚒  comment, let me know what you think.

Other than that 🚢 🔥 🔥 🔥

File: daemon.py (hunk 1 above)
Comment (degemer) [inline daemon.py]: Indeed, it works for both the collector and dogstatsd, I'm going to let this one and remove the duplicate in the collector (`Running in foreground`).

**Comment by degemer** [comment]
Done, thanks @olivielpeau !
Will merge once it's 💚 .

**Pull request merged**